lib/usd/hdMaya/delegates/sceneDelegate.cpp

```diff
@@ -503,6 +503,35 @@ HdMayaMaterialAdapterPtr HdMayaSceneDelegate::GetMaterialAdapter(const SdfPath&
     return iter == _materialAdapters.end() ? nullptr : iter->second;
 }
 
+template <typename AdapterPtr, typename Map>
+AdapterPtr HdMayaSceneDelegate::Create(const MDagPath& dag,
+    const std::function<AdapterPtr(HdMayaDelegateCtx*, const MDagPath&)>& adapterCreator,
+    Map& adapterMap, bool isSprim)
+{
+    if (!adapterCreator) {
+        return {};
+    }
+
+    TF_DEBUG(HDMAYA_DELEGATE_INSERTDAG)
+        .Msg(
+            "HdMayaSceneDelegate::InsertDag::"
```

Comment:
Assuming this should be changed to `"HdMayaSceneDelegate::Create::"`?

Reply:
Sure can add it.
I am curious if anyone is using the debug-message logging features?
I never do, and personally find it a bit harder to read the code with them...

Reply:
Oh, I definitely find them very useful!  The targeted feedback is great - what you get if you just do `TF_DEBUG=*` can be overwhelming (though occasionally still useful too).